python/ql/src/experimental/semmle/python/frameworks/Log.qll

```diff
@@ -0,0 +1,92 @@
+/**
+ * Provides classes modeling security-relevant aspects of the log libraries.
+ */
+
+private import python
+private import semmle.python.dataflow.new.DataFlow
+private import semmle.python.dataflow.new.TaintTracking
+private import semmle.python.dataflow.new.RemoteFlowSources
+private import experimental.semmle.python.Concepts
+private import semmle.python.frameworks.Flask
+private import semmle.python.ApiGraphs
+
+/**
+ * Provides models for Python's log-related libraries.
+ */
+private module log {
+  /**
+   * Log output method list.
+   *
+   * See https://docs.python.org/3/library/logging.html#logger-objects
+   */
+  private class LogOutputMethods extends string {
+    LogOutputMethods() { this in ["info", "error", "warn", "warning", "debug", "critical"] }
```

Comment:
We seem to be missing "debug", "log" , and "exception" from this list.

Reply:
Pr updated.